### ISSUE TITLE
no work any more ?

### ISSUE THREAD

**Issue by djmisterjon** [kisstkondoros/gutter-preview#41]
hi friend, i see you add some commit, i dont know since when but in my **.js** file , when i link image , it seem not work anymore ?

did you have tips ?
thanks your plugin are awsome.

![image](https://user-images.githubusercontent.com/24865815/53252386-c4684580-368c-11e9-83ff-83b280916f40.png)

**EDIT:** i also just try disable all extentions and it seem not work...

**Comment by usernamehw** [comment]
```
 ERR Cannot convert undefined or null to object: TypeError: Cannot convert undefined or null to object
	at Function.keys (<anonymous>)
	at loadPathsFromTSConfig (.vscode\extensions\kisstkondoros.vscode-gutter-preview-0.17.4\out\src\extension.js:32:20)
	at symbolUpdater (.vscode\extensions\kisstkondoros.vscode-gutter-preview-0.17.4\out\src\extension.js:96:35)
	at scan (.vscode\extensions\kisstkondoros.vscode-gutter-preview-0.17.4\out\src\decorator.js:138:13)
	at Timeout.throttleIds.(anonymous function).setTimeout [as _onTimeout] (.vscode\extensions\kisstkondoros.vscode-gutter-preview-0.17.4\out\src\decorator.js:19:17)
	at ontimeout (timers.js:427:11)
	at tryOnTimeout (timers.js:289:5)
	at listOnTimeout (timers.js:252:5)
	at Timer.processTimers (timers.js:212:10)
```

**Comment by kisstkondoros** [comment]
Thanks for the issue report and for the error log! The issue should be fixed in 0.17.5, sorry for the inconvenience.
